iati_sphinx_theme/locale/es/LC_MESSAGES/sphinx.po

```diff
@@ -0,0 +1,93 @@
+# Spanish translations for PROJECT.
+# Copyright (C) 2024 ORGANIZATION
+# This file is distributed under the same license as the PROJECT project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2024.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PROJECT VERSION\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2024-11-12 16:24+0200\n"
+"PO-Revision-Date: 2024-11-12 16:24+0200\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language: es\n"
+"Language-Team: es <LL@li.org>\n"
```

Comment:
Can we remove this email address?

Reply:
I can't find a way to remove just this line, as the file is autogenerated, so if we delete it manually it will just overwrite it again next time the command is run